.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# See: https://help.github.com/articles/about-codeowners/
+
+# These owners will be the default owners for everything in the repo.
+*       @daneshk @MadhukaHarith92 @sahanHe
```

Comment:
Seems like we need commitership to add as code owner

Reply:
Updated in [944beb6](https://github.com/ballerina-platform/persist-tools/pull/12/commits/944beb65051d7602683cebf5d4807854377bdf8a).